frontend/src/app/application-forms/fields/file-upload.component.html

```diff
@@ -1,14 +1,26 @@
-<div class="usa-file-input" [ngClass]="{'file-selected ng-valid': uploader.queue[0], 'ng-untouched': !uploader.queue[0], 'ng-invalid': errorMessage, 'required': required}">
-  <button (click)="clickInput($event)" [attr.aria-label]="!field.value ? 'click here to upload a file for ' + name: 'click here to replace file for ' + name" class="usa-button upload-file-btn" [ngClass]="{'usa-button-secondary-alt': uploader.queue[0]}" aria-live="polite" [attr.aria-required]="required ? 'true' : null">
-    <span id="{{type}}-choose-file" *ngIf="!uploader.queue[0] && !field.value">Choose file</span>
-    <span id="{{type}}-replace-file" *ngIf="uploader.queue[0] || field.value">Replace<span class="hidden-text"> file </span></span>
-    <span class="hidden-text">to upload for {{name}}</span>
+<div class="usa-file-input" [ngClass]="{'ng-valid': uploader.queue[0], 'ng-untouched': !uploader.queue[0], 'ng-invalid': errorMessage, 'required': required}">
+  <div class="upload-queue" *ngIf="uploader.queue[0] || field.value">
+    <ng-template class="file-item" ngFor let-item [ngForOf]="uploader.queue" let-i="index" [ngForTrackBy]="trackByFn">
+      <div class="upload-item">
+        <div id="{{type}}-file-name" class="file-name" *ngIf="uploader.queue[0] || field.value">{{uploader.queue[i].file.name}}</div>
+        <div class="file-upload-buttons">
+          <button (click)="clickInput($event, 'Delete', i)" [attr.aria-label]="!field.value ? 'click here to upload a file for ' + name: 'click here to replace file for ' + name" class="usa-button usa-button-red-alt usa-delete-button" aria-live="polite" [attr.aria-required]="required ? 'true' : null">Delete</button>
+          <button (click)="clickInput($event, 'Replace', i)" [attr.aria-label]="!field.value ? 'click here to upload a file for ' + name: 'click here to replace file for ' + name" class="usa-button usa-button-secondary-alt usa-replace-button" aria-live="polite" [attr.aria-required]="required ? 'true' : null">Replace</button>
+        </div>
+      </div>
+    </ng-template>
+  </div>
+
+  <button (click)="clickInput($event,'Upload', i)" [attr.aria-label]="!field.value ? 'click here to upload a file for ' + name: 'click here to replace file for ' + name" class="usa-button upload-file-btn" aria-live="polite" [attr.aria-required]="required ? 'true' : null">
+    <span id="{{type}}-choose-file" *ngIf="!uploader.queue[0]">Upload file</span>
+    <span id="{{type}}-choose-file" *ngIf="uploader.queue[0]">Upload  Another</span>
```

Comment:
Please remove the extra space between `Upload` and `Another`

Reply:
@briandavidson - done!